tests/Perf/Microsoft.Identity.Web.Perf.Client/TokenCacheHelper.cs

```diff
@@ -14,29 +14,21 @@ static class TokenCacheHelper
         /// </summary>
         public static readonly string s_cacheFilePath = System.Reflection.Assembly.GetExecutingAssembly().Location + ".msalcache.bin3";
 
-        private static readonly object s_fileLock = new object();
-
         public static void BeforeAccessNotification(TokenCacheNotificationArgs args)
         {
```

Comment:
The lock is not necessary and takes time

Reply:
yes, i already removed that